### PR TITLE
Updated pipelines to use the RP-Config for Ev2 Manifest Generation

### PR DESCRIPTION
**Pull request by coreyperkins**
### Which issue this PR addresses: / ### What this PR does / why we need it:

Checked out RP-Config for mirroring and the bootstrapper. 
Added RP_CONFIG_PATH to the go Ev2 manifest generation. RP_CONFIG_PATH is used to point to the RP-Config from the Ev2 Generator to generate the Ev2 manifests

### Test plan for issue:

Ran the Ev2 Generator and completed a deployment.

### Is there any documentation that needs to be updated for this PR?

No, this process is already documented.

### CODE REVIEW

File: .pipelines/onebranch/pipeline.bootstrapper.pullrequest.yml [hunk 1]
```diff
@@ -13,23 +13,26 @@ pr: none
...
+  LinuxContainerImage: cdpxlinux.azurecr.io/global/ubuntu-1804-all:5.0 # Docker image which is used to build the project https://aka.ms/obpipelines/containers
   Debian_Frontend: noninteractive
 
 parameters:
-- name: buildFluentBit
-  displayName: Build Fluent Bit?
-  type: boolean
+  - name: buildFluentBit
+    displayName: Build Fluent Bit?
+    type: boolean
```
Comment (troy0820) [inline .pipelines/onebranch/pipeline.bootstrapper.pullrequest.yml:22]: nit: While still valid yaml, this whitespace change isn't necessary and makes the review more tedious from what actually chaned.

Comment (troy0820) [reply .pipelines/onebranch/pipeline.bootstrapper.pullrequest.yml:22]: also nit: Is the `?` necessary in the display name for the `Build Fluent Bit?`

File: .pipelines/onebranch/pipeline.bootstrapper.pullrequest.yml [hunk 2]
```diff
@@ -13,23 +13,26 @@ pr: none
...
+    - repository: templates
+      type: git
+      name: OneBranch.Pipelines/GovernedTemplates
+      ref: refs/heads/main
+    - repository: rhado
+      type: git
+      name: ARO.Pipelines
+    - repository: rpconfig
+      type: git
+      name: RP-Config
```
Comment (troy0820) [inline .pipelines/onebranch/pipeline.bootstrapper.pullrequest.yml:35]: nit: I think the editor on save may have added white space to this.  Same nit as above with the whitespace.  
_note: still valid yaml_

**Review by troy0820 — COMMENTED** [review]
A couple of nits but the building of this and deployment history would be good to verify that this change worked in the intended environment.

File: .pipelines/onebranch/pipeline.bootstrapper.pullrequest.yml (hunk 2 above)
Comment (coreyperkins) [reply .pipelines/onebranch/pipeline.bootstrapper.pullrequest.yml:35]: @troy0820 I'll take care of this shortly, thanks!

**Pull request closed without merging**